proposals/0000-add-support-for-css-color-function.md

```diff
@@ -0,0 +1,109 @@
+          ')'
+      ),
+```
+
+2. Include the color space in the return value of [StyleSheet processColor](https://github.com/facebook/react-native/blob/63213712125795ac082597dad2716258b90cdcd5/packages/react-native/Libraries/StyleSheet/processColor.js) and [Animated processColor](https://github.com/facebook/react-native/blob/main/packages/react-native/Libraries/Animated/nodes/AnimatedColor.js)
+
+```js
+return { ["display-p3"]: true, r, g, b, a };
+```
+
+### iOS Changes
+
+1. Update [RCTConvert](https://github.com/facebook/react-native/blob/781b637db4268ad7f5f3910d99ebb5203467840b/packages/react-native/React/Base/RCTConvert.m#L881) to handle setting new color values for color space. If color space is not included preserve existing behavior.
+
+2. Update [RCTConversions](https://github.com/facebook/react-native/blob/16ad818d21773cdf25156642fae83592352ae534/packages/react-native/React/Fabric/RCTConversions.h#L37) to handle setting new color values for color space. If color space is not included preserve existing behavior.
+
+3. Update [RCTTextPrimitivesConversions](https://github.com/facebook/react-native/blob/ac1cdaa71620d5bb4860237cafb108f6aeae9aef/packages/react-native/ReactCommon/react/renderer/textlayoutmanager/platform/ios/react/renderer/textlayoutmanager/RCTTextPrimitivesConversions.h#L116) to handle setting new color values for color space. If color space is not included preserve existing behavior.
+
+```objc
+[UIColor colorWithDisplayP3Red:components.red green:components.green blue:components.blue alpha:components.alpha]
+```
+
+### Android Changes
+
+1. Enable the wide color gamut in [ReactActivity](https://github.com/facebook/react-native/blob/7625a502960e6b107e77542ff0d6f40fbf957322/packages/react-native/ReactAndroid/src/main/java/com/facebook/react/ReactActivity.java#L22) if available.
+
+```java
+Window window = getWindow();
+boolean isScreenWideColorGamut = window.getDecorView().isScreenWideColorGamut()
+if (isScreenWideColorGamut) {
+  window.setColorMode(PixelFormat.COLOR_MODE_WIDE_COLOR_GAMUT)
+}
+```
+
+2. Update [ColorUtil](https://github.com/facebook/react-native/blob/a6964b36294c3bfea09c0cdd65c5d0e3949f2dae/packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/view/ColorUtil.java#L17) and [ColorPropConverter](https://github.com/facebook/react-native/blob/781b637db4268ad7f5f3910d99ebb5203467840b/packages/react-native/ReactAndroid/src/main/java/com/facebook/react/bridge/ColorPropConverter.java#L27) to return new color values for color space as longs.
+
+```java
+@ColorLong long p3 = pack(1.0f, 1.0f, 0.0f, 1.0f, ColorSpace.Named.DISPLAY_P3);
+Color opaqueYellow = Color.valueOf(p3);
+```
+
+### Docs
+
+Update the [Color Reference](https://github.com/facebook/react-native-website/blob/main/docs/colors.md) to document color function usage in React Native.
+
+## Drawbacks
+
+There should be no breaking changes for users but the color implementation will be a bit more complicated than before.
```

Comment:
We need a little bit more details on why there will be no breaking changes, probably split by platforms. I.e.: for iOS, if the color space is not specified, we will continue using the default `UIColor` with RGB values constructor.